model-optimizer/mo/utils/versions_checker.py

```diff
@@ -44,34 +44,74 @@ def check_python_version():
         return 1
 
 
-def parse_versions_list(required_fw_versions, version_list):
+def parse_versions_list(required_fw_versions, version_list, env_params):
     """
     Please do not add parameter type annotations (param:type).
     Because we import this file while checking Python version.
     Python 2.x will fail with no clear message on type annotations.
 
-    Parsing requirements versions
+    Parsing requirements versions for a dependency,
+    if environment version (python_version, etc.) is satisfied
     :param required_fw_versions: String with fw versions from requirements file
     :param version_list: List for append
+    :param env_params: a dictionary of environment variables
```

Comment:
nit: it is not actually 'environment variables'. It is a python version in our case...

Reply:
I am ok to rename this dictionary. It can be also a platform used for markers.

Reply:
I have renamed it to env_setup and now there is no mention about env vars